spec/system/jobseekers/jobseekers_can_manage_their_job_alerts_from_the_email_spec.rb

```diff
@@ -69,11 +69,8 @@
         click_on I18n.t("buttons.update_alert")
       end
 
-      it "shows the confirmation page" do
```

Comment:
Combined this test with the following one because it just asserts presence of a confirmation banner but it meant this before block was being run twice:

```
before do
        message_delivery = instance_double(ActionMailer::MessageDelivery)
        expect(Jobseekers::SubscriptionMailer).to receive(:update) { message_delivery }
        expect(message_delivery).to receive(:deliver_later)

        update_subscription_fields
        click_on I18n.t("buttons.update_alert")
      end
```

Reply:
Non-blocking suggestion:

Now that this is a single tests, we could just wrap everything within the `it "...` rather than splitting it between `before .... it...`, especially when the `before` contains `expect` calls (that is something that should be triggering `rubocop-rspec` errors :no_good: